<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomoToolConfigSavu</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Savu configuration</string>
  </property>
  <widget class="QWidget" name="savuConfigCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QMenuBar" name="menubar_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>21</height>
           </size>
          </property>
          <widget class="QMenu" name="menuFile">
           <property name="title">
            <string>File</string>
           </property>
           <addaction name="separator"/>
          </widget>
          <addaction name="menuFile"/>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitterPlugins">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QFrame" name="pluginBar">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="lblFilename">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Available Plugins:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>284</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnTransfer">
                    <property name="font">
                     <font>
                      <pointsize>26</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>→</string>
                    </property>
                    <property name="autoDefault">
                     <bool>false</bool>
                    </property>
                    <property name="default">
                     <bool>false</bool>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listAvailablePlugins">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>3</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="availablePluginDesc">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>120</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QFrame" name="currentBar">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Current configuration</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>239</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnMoveUp">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                      <weight>75</weight>
                      <italic>false</italic>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>▲</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnMoveDown">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                      <weight>75</weight>
                      <italic>false</italic>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>▼</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnRemove">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>X</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QTreeWidget" name="treeCurrentPlugins">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Here you can modify the parameters of the plugins.
Double-click on them to modify their values.</string>
                 </property>
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="currentPluginDesc">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>120</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Help">
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>728</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
